stdlib/public/Python/PythonLibrary.swift

```diff
@@ -82,9 +84,13 @@ public struct PythonLibrary {
   }
 }
 
-// Methods of `PythonLibrary` required to set a given the Python version.
+// Methods of `PythonLibrary` required to set a given Python version.
 public extension PythonLibrary {
   static func useVersion(_ major: Int, _ minor: Int? = nil) {
+    precondition(!librarySymbolsLoaded, """
+      Error: \(#function) should not be called after any Python library
+      symbols have already been loaded.
```

Comment:
Could you please remove “symbols” from this sentence? It is not a directly user-facing term. The sentence without it would carry the same accurate meaning.

Reply:
Good call. :thumbsup: